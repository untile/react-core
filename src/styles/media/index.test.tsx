/**
 * Module dependencies.
 */

import { media } from '.';
import { render } from '@testing-library/react';
import React from 'react';
import styled from 'styled-components';

/**
 * `Wrapper` styled component.
 */

const Wrapper = styled.div`
  ${media.min.xl`
    background-color: red;
  `}
`;

/**
 * Test `svgBackground` style util.
 */

describe(`'media' style util`, () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Wrapper />);

    expect(asFragment()).toMatchSnapshot();
  });
});
