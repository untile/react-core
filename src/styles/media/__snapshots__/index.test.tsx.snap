// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'media' style util matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-beqWaB jXdjaq"
  />
</DocumentFragment>
`;
